app/services/trainees/create_from_hesa.rb

```diff
@@ -55,6 +56,24 @@ def provider_attributes
       provider ? { provider: provider } : {}
     end
 
+    def withdrawal_attributes
+      return {} unless trainee_withdrawn?
+
+      {
+        withdraw_date: hesa_trainee[:end_date],
+        withdraw_reason: reason_for_leaving,
+        state: "withdrawn",
```

Comment:
We might want to handle states all together at some point - but this is the only one we're setting for now from what I can see.